app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/SearchResultsActivity.java

```diff
@@ -43,4 +53,31 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return true;
     }
+
+
+    private void loadPersons() {
+        // get multiple persons
+        String json = loadJSONFromAsset("dummyData.json");
+        Persons persons = new Gson().fromJson(json, Persons.class);
```

Comment:
The class year shows up as 0 when it is null, which isn't quite correct. Gson thankfully handles these cases of the data being imperfect but not always in an ideal way. For the class year, change it to some special string when the value is 0/null (perhaps something like `n/a` or just blank), and try to do the same for the other `Person` fields.